pkg/apis/settings/settings.go

```diff
@@ -52,8 +54,8 @@ var defaultSettings = &Settings{
 // +k8s:deepcopy-gen=true
 type Settings struct {
 	AssumeRoleARN              string
-	AssumeRoleDuration         time.Duration `validate:"min=15m"`
-	ClusterName                string        `validate:"required"`
+	AssumeRoleDuration         *metav1.Duration `validate:"min=15m"`
```

Comment:
Why do we need to turn this into a pointer? Seems like all the rest of the settings are primitives.